### DISCUSSION TITLE
[Feature request] Batch Tag Editing

### DISCUSSION THREAD

**Discussion by ghost** [zanllp/infinite-image-browsing#454] [Ideas]
First of all, thanks for this fantastic tool.

To the point: Is it possible implement an _Assign Tag To Selection_ or some kind of batch system for tag management?

Example: I select many images, right click>Toggle Tag Selection>Like, then all selected images receive the "Like" tag.

Thanks again and keep the good work!

**Comment by zanllp** [comment]
Already supported.
https://github.com/zanllp/sd-webui-infinite-image-browsing/wiki/Change-log
